components/suggestion/search_channel_suggestion/search_channel_suggestion.tsx

```diff
@@ -68,14 +68,15 @@ function itemToName(item: Channel, currentUser: string): {icon: React.ReactEleme
 
 export default class SearchChannelSuggestion extends Suggestion {
     render(): JSX.Element {
-        const {item, isSelection, teammate, currentUser} = this.props;
+        const {isSelection, teammate, currentUser} = this.props;
+        const item = this.props.item as Channel;
 
         let className = 'suggestion-list__item';
         if (isSelection) {
             className += ' suggestion--selected';
         }
 
-        const nameObject = itemToName(item, currentUser);
+        const nameObject = itemToName(item, currentUser as string);
```

Comment:
Nit: Just curious why we need the cast for `currentUser`? Does it not think it's a `string`?

Reply:
No, it is optional in suggestion.tsx. (so its type here is string | undefined);
should this be written in an if statement instead?

Reply:
You can instead use a coalesce operator to pass in a blank string instead. Assuming that the method actually cares that it's an actual string and not a null.
```suggestion
        const nameObject = itemToName(item, currentUser ?? '');
```